README.md

```diff
@@ -1,11 +1,8 @@
-[![CharmHub Badge](https://charmhub.io/nginx-ingress-integrator/badge.svg)](https://charmhub.io/nginx-ingress-integrator)
-[![Publish to edge](https://github.com/canonical/nginx-ingress-integrator-operator/actions/workflows/publish_charm.yaml/badge.svg)](https://github.com/canonical/nginx-ingress-integrator-operator/actions/workflows/publish_charm.yaml)
-[![Promote charm](https://github.com/canonical/nginx-ingress-integrator-operator/actions/workflows/promote_charm.yaml/badge.svg)](https://github.com/canonical/nginx-ingress-integrator-operator/actions/workflows/promote_charm.yaml)
-[![Discourse Status](https://img.shields.io/discourse/status?server=https%3A%2F%2Fdiscourse.charmhub.io&style=flat&label=CharmHub%20Discourse)](https://discourse.charmhub.io)
+# Nginx Ingress Integrator Operator
 
 A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators) deploying and managing external access to HTTP/HTTPS services in a
```

Comment:
I think we should somehow distinguish this as an integrator charm(workload-less) and also how consumer charms could use it. It isn't very clear that this charm doesn't deploy an ingress